api/models/Group.js

```diff
@@ -3,6 +3,10 @@ import { clone, defaults, difference, flatten, intersection, isEmpty, map, merge
 import randomstring from 'randomstring'
 import wkx from 'wkx'
 import HasSettings from './mixins/HasSettings'
+import findOrCreateThread from './post/findOrCreateThread'
+import createComment from './comment/createComment'
```

Comment:
I don't see this function being used in this PR. Is it from something else?

Reply:
good catch